openapi/openapi.yaml

```diff
@@ -612,6 +612,7 @@ components:
       properties:
```

Comment:
A required field needs to go before an optional field. Could you please swap the order of the two properties? 

We require this because the generated python class for models will have required properties as positional arguments and optional properties as named arguments that default to `None`. A more detailed explanation can be found in #395.

This particular instance does not throw an error because our flask template doesn't actually generate any property for models that allow `additionalProperties` but is something we plan to fix (#432). Unfortunately, we will have to enforce this rule manually through code reviews before that issue is fixed.